### PR TITLE
Add clipboard action to Personal Settings Fed Cloud Id

### PR DESCRIPTION
**Pull request by rullzer**
For #735

CC: @MorrisJobke @nickvergessen @icewind1991 @schiessle @jancborchardt @LukasReschke

### CODE REVIEW

**Comment by schiessle** [comment]
works great! Is it by purpose that the cloud ID gets marked if I click on the copy button?

Just a small idea, I would consider to add a margin-left to the copy bottom so that it doesn't look to squeezed. Played around in my browser and like the result with 5px.

**Comment by rullzer** [comment]
Yes selecting it basically highlights what you copy. And it makes it easier to directly copy in the case of a failed copy (for some reason).

**Comment by rullzer** [comment]
I'll change the padding

**Comment by MorrisJobke** [comment]
```
position: relative;
top: 3px;
```

Before:
bildschirmfoto 2016-08-09 um 14 15 39
After:
bildschirmfoto 2016-08-09 um 14 15 34

**Comment by MorrisJobke** [comment]
beside that it works nice 👍

**Comment by rullzer** [comment]
Used both comments:

Before:
before

After:
after

**Comment by schiessle** [comment]
looks and works great now :+1:

File: apps/federatedfilesharing/templates/settings-personal.php [hunk 1]
```diff
@@ -15,7 +15,8 @@
 
 		<p>
 			<?php p($l->t('Your Federated Cloud ID:')); ?>
-			<strong><?php p($_['cloudId']); ?></strong>
+			<strong id="cloudid"><?php p($_['cloudId']); ?></strong>
+			<a class="clipboardButton icon icon-clippy" data-clipboard-target="#cloudid"></a>
```
Comment (jancborchardt) [inline apps/federatedfilesharing/templates/settings-personal.php]: The extra `.icon` class is not needed. ;)

**Pull request merged**